src/test/lib/operators/join_test_runner.cpp

```diff
@@ -758,6 +784,18 @@ TEST_P(JoinTestRunner, TestJoin) {
       indexed_used_count = range_end - range_begin;
     }
 
+    if (indexed_input.index_scope == IndexScope::Table) {
+      if (indexed_input.table_type == InputTableType::Data && configuration.secondary_predicates.empty() &&
```

Comment:
`configuration.secondary_predicates` does not have to be checked since the check `JoinOperator::supports(support_configuration)` in line 305 already ensures that the index join is only used when only a primary predicate is present.